packages/page-statistics/src/gql/proposals.graphql

```diff
@@ -1,5 +1,5 @@
-query Proposals {
-  proposalItems {
+query Proposals($PerPage: Int!, $Offset: Int!) {
+  proposalItems(first: $PerPage, offset: $Offset) {
```

Comment:
Can we add relevant documentation comments for each of these queries (e.g. expected response and usage example). Unless they are documented elsewhere.